cli/command/image/formatter_history.go

```diff
@@ -82,10 +82,16 @@ func (c *historyContext) CreatedAt() string {
 	return time.Unix(c.h.Created, 0).Format(time.RFC3339)
 }
 
+// epoch is the time before which created-at dates are not displayed with human units.
+var epoch = time.Date(2000, 1, 1, 0, 0, 0, 0, time.UTC).Unix()
```

Comment:
Anything before [March 15, 2013](https://pyvideo.org/pycon-us-2013/the-future-of-linux-containers.html) would be unlikely, so I guess we're safe with 2000 yes 😄

Reply:
I've got several container images where I intentionally set the date to something meaningful but still before 2013 -- how many of these "really old" dates are there?  Would it be tenable to have a small list of exceptions instead of hard-coding a specific "cut-off" date?

(Also for consistency, shouldn't it be replaced with `N/A` instead of the empty string?)

Reply:
Why do we need to hide anything but zero value at all? What is the case where non-zero value is set but should be not shown?

Reply:
> Why do we need to hide anything but zero value at all? What is the case where non-zero value is set but should be not shown?

Some builders use 1980 as the magical reproducibility date (see https://github.com/docker/cli/issues/2995#issuecomment-790055255). Other than that I don't know any other magic values that denote special dates, but there may be more.

Rather than interpreting this change as "hide special dates chosen for reproducibility" I think this change is more about hiding values of "created" that clearly don't represent the actual date the image was created. If you actually created a Docker image on July 3, 1985, it's only because you had access to a time machine, which you're using for very mundane purposes. 😆

That being said, I'm happy to make whatever change to this will upset the fewest folks. The only date I really care about is 1/1/1970 (and 1980 for the buildpacks folks' sake). If the consensus is that we should special-case those dates, that's okay too.

(I can also change this to `N/A` if that's the consensus)

Thanks for merging this @thaJeztah ! 🙏

Reply:
> it's only because you had access to a time machine

You don't need a time machine to set that date. Any image author can set this date to any value they like. My assumption is that if they set it to some weird value they had some reason for it, and that reason can't be that they didn't want this date to ever show up because then they would have used zero value instead. And if they accidentally set it to a weird value it is also beneficial to show it so that they can notice the bug and fix it.

Reply:
Fair enough.

However, if they set a specific meaningful date, they presumably want to see that the date is _exactly_ that date and not somewhere near that date. In that case, they probably want to check the full timestamp string, instead of `37 years ago`, which will become `38 years ago` eventually. Displaying the full timestamp string is unchanged by this PR.

Reply:
FWIW, 2000-01-01 was just a date I picked out of whimsy for the epoch, and has no specific importance other than that it looked like a good date, based on the assumption that any legacy epoch dates of interest would likely fall before then.  We could easily just exempt 1970-01-01, 1980-01-01 and other epoch dates if people would rather do that :)